soql-analyzer/src/test/scala/com/socrata/soql/types/TestFunctions.scala

```diff
@@ -34,14 +34,16 @@ object TestFunctions {
 
   val Max = f("max", FunctionName("max"), Map("a" -> Ordered), Seq(VariableType("a")), Seq.empty, VariableType("a"), isAggregate = true)
   val Sum = f("sum", FunctionName("sum"), Map("a" -> NumLike), Seq(VariableType("a")), Seq.empty, VariableType("a"), isAggregate = true)
-  val Avg = f("avg", FunctionName("avg"),
-              Map("a" -> Set(TestNumber), "?" -> AllTypes),
-              Seq(VariableType("a")),
-              Seq(VariableType("?")),
-              VariableType("a"),
-              isAggregate = true)
+  val Avg = f("avg", FunctionName("avg"), Map("a" -> NumLike), Seq(VariableType("a")), Seq.empty, VariableType("a"), isAggregate = true)
   val CountStar = mf("count(*)", SpecialFunctions.StarFunc("count"), Seq(), Seq.empty, TestNumber, isAggregate = true)
 
+  val WindowFunctionOver = f("wf_over",
+                             FunctionName("wf_over"),
```

Comment:
A name from `SpecialFunctions` would seem appropriate here, if you don't want people actually writing `wf_over(...)`

Reply:
(that'd also provide a reasonable hook for printing it back as window-function syntax)